<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>GxE Fam Project</title>
  <meta name="description" content="GxE Fam Project">
  <meta name="generator" content="bookdown 0.7 and GitBook 2.6.7">

  <meta property="og:title" content="GxE Fam Project" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="GxE Fam Project" />
  
  
  

<meta name="author" content="Andrey Ziyatdinov">


<meta name="date" content="2018-08-24">

  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="index.html">
<link rel="next" href="supplementary-material.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />







<meta name="robots" content="noindex">



<link rel="stylesheet" href="css/style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="chapter" data-level="1" data-path="manuscript.html"><a href="manuscript.html"><i class="fa fa-check"></i><b>1</b> Manuscript</a><ul>
<li class="chapter" data-level="1.1" data-path="manuscript.html"><a href="manuscript.html#introduction"><i class="fa fa-check"></i><b>1.1</b> Introduction</a></li>
<li class="chapter" data-level="1.2" data-path="manuscript.html"><a href="manuscript.html#methods-and-analytical-derivations"><i class="fa fa-check"></i><b>1.2</b> Methods and Analytical Derivations</a><ul>
<li class="chapter" data-level="1.2.1" data-path="manuscript.html"><a href="manuscript.html#linear-mixed-model-for-genetic-association-study"><i class="fa fa-check"></i><b>1.2.1</b> Linear mixed model for genetic association study</a></li>
<li class="chapter" data-level="1.2.2" data-path="manuscript.html"><a href="manuscript.html#testing-marginal-genetic-effect"><i class="fa fa-check"></i><b>1.2.2</b> Testing marginal genetic effect</a></li>
<li class="chapter" data-level="1.2.3" data-path="manuscript.html"><a href="manuscript.html#testing-gene-environment-interaction-effect"><i class="fa fa-check"></i><b>1.2.3</b> Testing gene-environment interaction effect</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="manuscript.html"><a href="manuscript.html#results"><i class="fa fa-check"></i><b>1.3</b> Results</a><ul>
<li class="chapter" data-level="1.3.1" data-path="manuscript.html"><a href="manuscript.html#analytical-results-for-testing-marginal-genetic-effect"><i class="fa fa-check"></i><b>1.3.1</b> Analytical results for testing marginal genetic effect</a></li>
<li class="chapter" data-level="1.3.2" data-path="manuscript.html"><a href="manuscript.html#analytical-results-for-testing-gene-environment-interaction-effect"><i class="fa fa-check"></i><b>1.3.2</b> Analytical results for testing gene-environment interaction effect</a></li>
<li class="chapter" data-level="1.3.3" data-path="manuscript.html"><a href="manuscript.html#implication-in-association-studies"><i class="fa fa-check"></i><b>1.3.3</b> Implication in association studies</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="supplementary-material.html"><a href="supplementary-material.html"><i class="fa fa-check"></i><b>2</b> Supplementary Material</a><ul>
<li class="chapter" data-level="2.1" data-path="supplementary-material.html"><a href="supplementary-material.html#supplementary-figures"><i class="fa fa-check"></i><b>2.1</b> Supplementary Figures</a></li>
<li class="chapter" data-level="2.2" data-path="supplementary-material.html"><a href="supplementary-material.html#propositions"><i class="fa fa-check"></i><b>2.2</b> Propositions</a><ul>
<li class="chapter" data-level="2.2.1" data-path="supplementary-material.html"><a href="supplementary-material.html#quadratic-forms"><i class="fa fa-check"></i><b>2.2.1</b> Quadratic forms</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="supplementary-material.html"><a href="supplementary-material.html#derivations"><i class="fa fa-check"></i><b>2.3</b> Analytical derivations</a></li>
<li class="chapter" data-level="2.4" data-path="supplementary-material.html"><a href="supplementary-material.html#simulations"><i class="fa fa-check"></i><b>2.4</b> Simulations</a><ul>
<li class="chapter" data-level="2.4.1" data-path="supplementary-material.html"><a href="supplementary-material.html#unrelated-marginal-genetic-effect"><i class="fa fa-check"></i><b>2.4.1</b> Unrelated: marginal genetic effect</a></li>
<li class="chapter" data-level="2.4.2" data-path="supplementary-material.html"><a href="supplementary-material.html#families-marginal-genetic-effect"><i class="fa fa-check"></i><b>2.4.2</b> Families: marginal genetic effect</a></li>
<li class="chapter" data-level="2.4.3" data-path="supplementary-material.html"><a href="supplementary-material.html#unrelated-interaction-effect"><i class="fa fa-check"></i><b>2.4.3</b> Unrelated: interaction effect</a></li>
<li class="chapter" data-level="2.4.4" data-path="supplementary-material.html"><a href="supplementary-material.html#familes-two-genetic-components-interaction-effect"><i class="fa fa-check"></i><b>2.4.4</b> Familes (two genetic components): interaction effect</a></li>
<li class="chapter" data-level="2.4.5" data-path="supplementary-material.html"><a href="supplementary-material.html#familes-one-genetic-component-interaction-effect"><i class="fa fa-check"></i><b>2.4.5</b> Familes (one genetic component): interaction effect</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">GxE Fam Project</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="manuscript" class="section level1">
<h1><span class="header-section-number">1</span> Manuscript</h1>
<p><strong>Title</strong>: Statistical power in GWAS revisited: sample size, genetic relatedness, and gene-by-environment interactions</p>
<p><strong>Abstract</strong>:
Genome-wide association studies (GWAS) have identified
thousands of genetic variants associated with complex diseases and 
heavily rely on increasing the sample size.
Recent analyses of biobank-scale genetic data suggest:
(i) inclusion of genetically related individuals empowers GWAS <span class="citation">(Loh et al. <a href="#ref-loh2018mixed">2018</a>)</span>;
(ii) the wealth of collected environmental exposures has potential
to uncover gene-by-environmental interactions <span class="citation">(Young, Wauthier, and Donnelly <a href="#ref-young2016multiple">2016</a>)</span>.
However, quantification of GWAS power — 
the non-centrality parameter (<span class="math inline">\(NCP\)</span>) of association test,
which is proportional to the sample size (<span class="math inline">\(n\)</span>) and the variance explained by genetic variant (<span class="math inline">\(q^2\)</span>) — 
holds only for unrelated individuals.
Here, we first expanded it by incorporating individual relationships by linear mixed model.
We next studied gene-by-environment interactions,
where interaction effect on trait is tested in the presence of marginal genetic effect.
<!-- results/conclusions -->
In result, the derived formulas have a range of implications.
For testing marginal genetic effect, one can quickly assess the power in studies involving related individuals. Because of the potential gain in power for testing gene-by-environment interaction, the formula for interactions will allow optimization of the study design of related individuals.</p>
<div id="introduction" class="section level2">
<h2><span class="header-section-number">1.1</span> Introduction</h2>
</div>
<div id="methods-and-analytical-derivations" class="section level2">
<h2><span class="header-section-number">1.2</span> Methods and Analytical Derivations</h2>
<div id="linear-mixed-model-for-genetic-association-study" class="section level3">
<h3><span class="header-section-number">1.2.1</span> Linear mixed model for genetic association study</h3>
<span class="math display" id="eq:assoc">\[\begin{equation} 
  y \sim \mathcal{N} (\mu x_0 + \beta_g x_g, \sum{\sigma_i^2 R_i} + \sigma_r^2 I)
\tag{1.1}
\end{equation}\]</span>
<p>We showed in Section <a href="supplementary-material.html#derivations">2.3</a> that <span class="math inline">\(var(\hat{\beta}_g) = (x_g^T \hat{V}^{-1} x_g)^{-1}\)</span>, where <span class="math inline">\(\hat{V} = \sum{\hat{\sigma}_i^2 R_i} + \hat{\sigma}_r^2 I\)</span>.</p>
<p>We further consider <span class="math inline">\(x_g\)</span> as a realization of a random (vector) variable <span class="math inline">\(\mathcal{X}_g\)</span> with mean-vector of zeros and covariance matrix <span class="math inline">\(\Sigma_g = 2 p (1 - p) K\)</span>, where <span class="math inline">\(p\)</span> is the minor allele frequency of the genotype and <span class="math inline">\(K\)</span> is the kinship matrix in the general case.
As outlined in Section <a href="supplementary-material.html#propositions">2.2</a>, the following quadratic form <span class="math inline">\(\mathcal{X}_g^T A \mathcal{X}_g\)</span> is a scalar random variable given a transformation matrix <span class="math inline">\(A = \hat{V}^{-1}\)</span>,
and its mean is equal to <span class="math inline">\(E(\mathcal{X}_g^T A \mathcal{X}_g) = tr(A \Sigma_g)\)</span>.
Hence, that gives us the final expression <span class="math inline">\(var(\hat{\beta}_g) \approx (E(\mathcal{X}_g^T \hat{V}^{-1} \mathcal{X}_g))^{-1} = tr(\hat{V}^{-1} \Sigma_g)^{-1}\)</span>.</p>
</div>
<div id="testing-marginal-genetic-effect" class="section level3">
<h3><span class="header-section-number">1.2.2</span> Testing marginal genetic effect</h3>
<p>Consider a set of related individuals, which relatedness is expressed by the kinship matrix (<span class="math inline">\(K\)</span>), 
and association between outcome (<span class="math inline">\(y\)</span>) and genotype (<span class="math inline">\(g\)</span>) is evaluated by linear mixed model.
For a given phenotypic variance-covariance matrix (<span class="math inline">\(V\)</span>),
we show that the non-centrality parameter for marginal association test can be approximated as following:</p>
<span class="math display" id="eq:NCPrel">\[\begin{equation} 
  NCP = \hat{\beta}_g^2 / var(\hat{\beta}_g) \approx \hat{\beta}_g^2 tr( \hat{V}^{-1} \Sigma_g)
\tag{1.2}
\end{equation}\]</span>
<p> 
where <span class="math inline">\(tr\)</span> is the trace operator and <span class="math inline">\(\Sigma_g\)</span> is the covariance matrix of genotypes.</p>
<p>For families we have <span class="math inline">\(V = \sigma_k^2 K + \sigma_r^2 I\)</span> and <span class="math inline">\(\Sigma_g = \sigma_g^2 K\)</span>,
that results in following:</p>
<span class="math display" id="eq:NCPrelfam">\[\begin{equation} 
  NCP_{fam} = q^2 tr( (\sigma_k^2 K + \sigma_r^2 I)^{-1} K ) 
\tag{1.3}
\end{equation}\]</span>
<p>That means <span class="math inline">\(NCP_{rel} \leq NCP\)</span>, i.e. for the same sample size,
power is expected to decrease as relatedness between individuals increases —
which was previously shown analytically only for sib-pairs in (Visscher et al., 2008).</p>
</div>
<div id="testing-gene-environment-interaction-effect" class="section level3">
<h3><span class="header-section-number">1.2.3</span> Testing gene-environment interaction effect</h3>
<p>Further, we consider testing for fixed effect gene-by-environment interaction in related individuals. We have derived NCPrel+int=tr(V-1Σint), where Σint is the covariance matrix in distribution of the interaction variable. For families we have V= [h2K+I], Σint=q2w2Kint, where w2=f(1–f) and f is the frequency of binary exposure. Kint is a matrix derived from K and depends on realization of exposure in a given study. We argue that Kint is different from the matrix recently proposed in (Sul et al., 2016). We validated all results by simulations.</p>
<span class="math display" id="eq:NCPrelint">\[\begin{equation} 
  NCP_{rel+int} = \beta_{int}^2 tr( \hat{V}^{-1} \Sigma_{int})
\tag{1.4}
\end{equation}\]</span>
</div>
</div>
<div id="results" class="section level2">
<h2><span class="header-section-number">1.3</span> Results</h2>
<div id="analytical-results-for-testing-marginal-genetic-effect" class="section level3">
<h3><span class="header-section-number">1.3.1</span> Analytical results for testing marginal genetic effect</h3>
<table>
<caption><span id="tab:assoc-gen">Table 1.1: </span> Analytical comparison of study designs to detect marginal genetic association.
Study designs differ in individual relationships that informs modeling of outcome (<span class="math inline">\(y\)</span>) and distribution of genotype under association test (<span class="math inline">\(x_g\)</span>). The statistical power to detect an association depends on the non-centrality parameter <span class="math inline">\(NCP = \hat{\beta}_g^2 / var(\hat{\beta}_g)\)</span> and, thus, the formula for variance of <span class="math inline">\(\hat{\beta}_g\)</span> is reported for each study. Study designs under comparison include: unrelated individuals; unrelated individuals with a grouping factor such as house-hold (not related to a variable under test); unrelated individuals with modeling outcome conditioned on genetic effects using Genetic Relationship Matrix (GRM); related individuals in families. Notation: <span class="math inline">\(\tilde{x_g}\)</span>, mean-centered genotype vector <span class="math inline">\(x_g\)</span>; <span class="math inline">\(\delta_g^2 = 2 p (1 - p)\)</span>, the variance of genotype random variable with the minor allele frequency <span class="math inline">\(p\)</span>; <span class="math inline">\(K\)</span>, the additive kinship matrix for family-based study design.</caption>
<colgroup>
<col width="23%" />
<col width="15%" />
<col width="15%" />
<col width="46%" />
</colgroup>
<thead>
<tr class="header">
<th>Study design</th>
<th><span class="math inline">\(V = covar(y)\)</span></th>
<th><span class="math inline">\(\Sigma_g = covar(x_g)\)</span></th>
<th><span class="math inline">\(1 / var(\hat{\beta}_g)\)</span> from <span class="math inline">\(y \sim \mathcal{N}(\mu + \beta_g x_g, V)\)</span></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Unrelated</td>
<td><span class="math inline">\(\sigma_r^2 I\)</span></td>
<td><span class="math inline">\(\delta_g^2 I\)</span></td>
<td><span class="math inline">\(\tilde{x_g}^T \tilde{x_g} /\sigma_r^2 \approx \delta_g^2 N / \sigma_r^2\)</span></td>
</tr>
<tr class="even">
<td>Unrelated + Grouping</td>
<td><span class="math inline">\(\sigma_h^2 H + \sigma_r^2 I\)</span></td>
<td><span class="math inline">\(\delta_g^2 I\)</span></td>
<td><span class="math inline">\(\tilde{x_g}^T V^{-1} \tilde{x_g} \approx \delta_g^2 \mbox { } tr(V^{-1})\)</span></td>
</tr>
<tr class="odd">
<td>Unrelated + GRM</td>
<td><span class="math inline">\(\sigma_m^2 M + \sigma_r^2 I\)</span></td>
<td><span class="math inline">\(\delta_g^2 I\)</span></td>
<td><span class="math inline">\(\tilde{x_g}^T V^{-1} \tilde{x_g} \approx \delta_g^2 \mbox { } tr(V^{-1})\)</span></td>
</tr>
<tr class="even">
<td>Families</td>
<td><span class="math inline">\(\sigma_k^2 K + \sigma_r^2 I\)</span></td>
<td><span class="math inline">\(\delta_g^2 K\)</span></td>
<td><span class="math inline">\(\tilde{x_g}^T V^{-1} \tilde{x_g} \approx \delta_g^2 \mbox { } tr(V^{-1} K)\)</span></td>
</tr>
</tbody>
</table>
<div class="figure" style="text-align: center"><span id="fig:power-marginal"></span>
<img src="figures/07-figure-power-marginal-two-panels.png" alt="Two study designs with a structured variance components, unrelated individuals with a non-genetic grouping factor and related individuals in families, are compared to the reference study design of unrelated individuals. The performance is evaluated to detect marginal genetic effect under the model \(y \sim \mathcal{N}(\mu + \beta_g x_g, V)\) (see also Table 1.1). To make the study designs comparable, the sum of variance components in the \(V\) matrix is equal to one. (a) The effective size multiplier, estimated as \(tr(V^{-1} \Sigma_g) / n\), depends on the variance explained. (b) When the variance explained fixed to 50% and the sample size varies, Notation: \(n\), the sample size." width="90%" />
<p class="caption">
Figure 1.1: Two study designs with a structured variance components, unrelated individuals with a non-genetic grouping factor and related individuals in families, are compared to the reference study design of unrelated individuals. The performance is evaluated to detect marginal genetic effect under the model <span class="math inline">\(y \sim \mathcal{N}(\mu + \beta_g x_g, V)\)</span> (see also Table <a href="manuscript.html#tab:assoc-gen">1.1</a>). To make the study designs comparable, the sum of variance components in the <span class="math inline">\(V\)</span> matrix is equal to one. (a) The effective size multiplier, estimated as <span class="math inline">\(tr(V^{-1} \Sigma_g) / n\)</span>, depends on the variance explained. (b) When the variance explained fixed to 50% and the sample size varies, Notation: <span class="math inline">\(n\)</span>, the sample size.
</p>
</div>

<p>Ref <a href="manuscript.html#fig:power-marginal">1.1</a></p>
</div>
<div id="analytical-results-for-testing-gene-environment-interaction-effect" class="section level3">
<h3><span class="header-section-number">1.3.2</span> Analytical results for testing gene-environment interaction effect</h3>
<div class="figure" style="text-align: center"><span id="fig:power-interaction"></span>
<img src="figures/08-figure-power-interaction-two-panels.png" alt="Comparison of study designs to detect interaction effect." width="75%" />
<p class="caption">
Figure 1.2: Comparison of study designs to detect interaction effect.
</p>
</div>

<p>Ref <a href="manuscript.html#fig:power-interaction">1.2</a></p>
</div>
<div id="implication-in-association-studies" class="section level3">
<h3><span class="header-section-number">1.3.3</span> Implication in association studies</h3>

</div>
</div>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-loh2018mixed">
<p>Loh, Po-Ru, Gleb Kichaev, Steven Gazal, Armin P Schoech, and Alkes L Price. 2018. “Mixed-Model Association for Biobank-Scale Datasets.” <em>Nature Genetics</em>. Nature Publishing Group, 1.</p>
</div>
<div id="ref-young2016multiple">
<p>Young, Alexander I, Fabian Wauthier, and Peter Donnelly. 2016. “Multiple Novel Gene-by-Environment Interactions Modify the Effect of Fto Variants on Body Mass Index.” <em>Nature Communications</em> 7. Nature Publishing Group:12724.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="index.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="supplementary-material.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"linkedin": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"download": null,
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(src))
      src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
