<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>GxE Fam Project</title>
  <meta name="description" content="GxE Fam Project">
  <meta name="generator" content="bookdown  and GitBook 2.6.7">

  <meta property="og:title" content="GxE Fam Project" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="GxE Fam Project" />
  
  
  

<meta name="author" content="Andrey Ziyatdinov">


<meta name="date" content="2018-09-30">

  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="index.html">
<link rel="next" href="supplementary-material.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />







<meta name="robots" content="noindex">



<link rel="stylesheet" href="css/style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="chapter" data-level="1" data-path="manuscript.html"><a href="manuscript.html"><i class="fa fa-check"></i><b>1</b> Manuscript</a><ul>
<li class="chapter" data-level="1.1" data-path="manuscript.html"><a href="manuscript.html#introduction"><i class="fa fa-check"></i><b>1.1</b> Introduction</a></li>
<li class="chapter" data-level="1.2" data-path="manuscript.html"><a href="manuscript.html#methods-and-analytical-derivations"><i class="fa fa-check"></i><b>1.2</b> Methods and Analytical Derivations</a><ul>
<li class="chapter" data-level="1.2.1" data-path="manuscript.html"><a href="manuscript.html#linear-mixed-model-for-genetic-association-study"><i class="fa fa-check"></i><b>1.2.1</b> Linear mixed model for genetic association study</a></li>
<li class="chapter" data-level="1.2.2" data-path="manuscript.html"><a href="manuscript.html#testing-marginal-genetic-effect"><i class="fa fa-check"></i><b>1.2.2</b> Testing marginal genetic effect</a></li>
<li class="chapter" data-level="1.2.3" data-path="manuscript.html"><a href="manuscript.html#testing-gene-environment-interaction-effect"><i class="fa fa-check"></i><b>1.2.3</b> Testing gene-environment interaction effect</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="manuscript.html"><a href="manuscript.html#results"><i class="fa fa-check"></i><b>1.3</b> Results</a><ul>
<li class="chapter" data-level="1.3.1" data-path="manuscript.html"><a href="manuscript.html#analytical-results-for-testing-marginal-genetic-effect"><i class="fa fa-check"></i><b>1.3.1</b> Analytical results for testing marginal genetic effect</a></li>
<li class="chapter" data-level="1.3.2" data-path="manuscript.html"><a href="manuscript.html#analytical-results-for-testing-gene-environment-interaction-effect"><i class="fa fa-check"></i><b>1.3.2</b> Analytical results for testing gene-environment interaction effect</a></li>
<li class="chapter" data-level="1.3.3" data-path="manuscript.html"><a href="manuscript.html#implication-in-association-studies"><i class="fa fa-check"></i><b>1.3.3</b> Implication in association studies</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="supplementary-material.html"><a href="supplementary-material.html"><i class="fa fa-check"></i><b>2</b> Supplementary Material</a><ul>
<li class="chapter" data-level="2.1" data-path="supplementary-material.html"><a href="supplementary-material.html#propositions"><i class="fa fa-check"></i><b>2.1</b> Propositions</a></li>
<li class="chapter" data-level="2.2" data-path="supplementary-material.html"><a href="supplementary-material.html#derivations"><i class="fa fa-check"></i><b>2.2</b> Analytical derivations</a><ul>
<li class="chapter" data-level="2.2.1" data-path="supplementary-material.html"><a href="supplementary-material.html#lmg"><i class="fa fa-check"></i><b>2.2.1</b> Testing marginal genetic effect in unrelated individuals</a></li>
<li class="chapter" data-level="2.2.2" data-path="supplementary-material.html"><a href="supplementary-material.html#lmmg"><i class="fa fa-check"></i><b>2.2.2</b> Testing marginal genetic effect in related individuals</a></li>
<li class="chapter" data-level="2.2.3" data-path="supplementary-material.html"><a href="supplementary-material.html#trfg"><i class="fa fa-check"></i><b>2.2.3</b> Effective size multiplier for testing marginal genetic effect</a></li>
<li class="chapter" data-level="2.2.4" data-path="supplementary-material.html"><a href="supplementary-material.html#lmge"><i class="fa fa-check"></i><b>2.2.4</b> Testing gene-environment interaction effect in unrelated individuals</a></li>
<li class="chapter" data-level="2.2.5" data-path="supplementary-material.html"><a href="supplementary-material.html#lmmge"><i class="fa fa-check"></i><b>2.2.5</b> Testing gene-environment interaction effect in related individuals</a></li>
<li class="chapter" data-level="2.2.6" data-path="supplementary-material.html"><a href="supplementary-material.html#trfge"><i class="fa fa-check"></i><b>2.2.6</b> Effective size multiplier for testing marginal gene-environment interaction effect</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="supplementary-material.html"><a href="supplementary-material.html#simulations"><i class="fa fa-check"></i><b>2.3</b> Simulations</a><ul>
<li class="chapter" data-level="2.3.1" data-path="supplementary-material.html"><a href="supplementary-material.html#unrelated-marginal-genetic-effect"><i class="fa fa-check"></i><b>2.3.1</b> Unrelated: marginal genetic effect</a></li>
<li class="chapter" data-level="2.3.2" data-path="supplementary-material.html"><a href="supplementary-material.html#families-marginal-genetic-effect"><i class="fa fa-check"></i><b>2.3.2</b> Families: marginal genetic effect</a></li>
<li class="chapter" data-level="2.3.3" data-path="supplementary-material.html"><a href="supplementary-material.html#unrelated-interaction-effect"><i class="fa fa-check"></i><b>2.3.3</b> Unrelated: interaction effect</a></li>
<li class="chapter" data-level="2.3.4" data-path="supplementary-material.html"><a href="supplementary-material.html#familes-two-genetic-components-interaction-effect"><i class="fa fa-check"></i><b>2.3.4</b> Familes (two genetic components): interaction effect</a></li>
<li class="chapter" data-level="2.3.5" data-path="supplementary-material.html"><a href="supplementary-material.html#familes-one-genetic-component-interaction-effect"><i class="fa fa-check"></i><b>2.3.5</b> Familes (one genetic component): interaction effect</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="supplementary-material.html"><a href="supplementary-material.html#supplementary-figures"><i class="fa fa-check"></i><b>2.4</b> Supplementary Figures</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">GxE Fam Project</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="manuscript" class="section level1">
<h1><span class="header-section-number">1</span> Manuscript</h1>
<p><strong>Title</strong>: Statistical power in GWAS revisited: effective sample size, genetic relatedness, and gene-by-environment interactions</p>
<p><strong>Title 2</strong>: Advantages and limitations of linear mixed model in genetic association studies</p>
<p><strong>Abstract</strong>: Genome-wide association studies (GWAS) have identified thousands of genetic variants associated with complex diseases and  heavily rely on increasing the sample size. Recent analyses of biobank-scale genetic data suggest: (i) inclusion of genetically related individuals empowers GWAS <span class="citation">(P.-R. Loh et al. <a href="#ref-loh2018mixed">2018</a>)</span>; (ii) the wealth of collected environmental exposures has potential to uncover gene-by-environmental interactions <span class="citation">(Young, Wauthier, and Donnelly <a href="#ref-young2016multiple">2016</a>)</span>. However, quantification of GWAS power —  the non-centrality parameter (<span class="math inline">\(NCP\)</span>) of association test, which is proportional to the sample size (<span class="math inline">\(n\)</span>) and the variance explained by genetic variant (<span class="math inline">\(q^2\)</span>) —  holds only for unrelated individuals. Here, we first expanded it by incorporating individual relationships by linear mixed model. We next studied gene-by-environment interactions, where interaction effect on trait is tested in the presence of marginal genetic effect. <!-- results/conclusions --> In result, the derived formulas have a range of implications. For testing marginal genetic effect, one can quickly assess the power in studies involving related individuals. Because of the potential gain in power for testing gene-by-environment interaction, the formula for interactions will allow optimization of the study design of related individuals.</p>
<div id="introduction" class="section level2">
<h2><span class="header-section-number">1.1</span> Introduction</h2>
<p>The vast majority of genome-wide association studies (GWAS) conducted so far has used standard fixed effect models. This strategy has been showed to be at the same time robust and fast, allowing for the analysis of thousands of individuals and millions of genetic variants in a reasonable computation time. However, in parallel, there has been increased interest in using linear mixed model (LMM) for the purpose of genome-wide association mapping. LMM has been considered for several reasons. Random effects models have primarily been used to control for the type I error rate account when analyzing related individuals. Indeed, when not modeled, genetic relationship will in general lead to an underestimation of the effect estimates, thus producing inflated statistics. LMM has also been showed to be of interest for increasing power, and because of that, there has been increasing literature about the possibility of using LMM more systematically in GWAS [ref]. However, LMM have also limitation, including in particular a dramatic increase in computation time, making the approach intractable for extremely large sample size, or when analyzing multiple phenotypes [ref]. … other issues …. Overall, many issues have been solved, and LMM will likely be increasingly used in GWAS settings by the community. However, the expected gain in power that one might achieve relative to the computational cost of LMM has not been solved.</p>
<p>Here, we proposed an analytical framework</p>
<p>Previous studies already discussed the ability of LMM to prevent false positive associations due to population stratification, how to reduce the computational burden of LMM [ref], and potential gain in power that might be achieved</p>
<p>However the later</p>
<p>How the 2 combine both…</p>
</div>
<div id="methods-and-analytical-derivations" class="section level2">
<h2><span class="header-section-number">1.2</span> Methods and Analytical Derivations</h2>
<div id="linear-mixed-model-for-genetic-association-study" class="section level3">
<h3><span class="header-section-number">1.2.1</span> Linear mixed model for genetic association study</h3>
<p>We consider the following linear mixed model to study the impact of relatedness among individuals on modeling a continuous phenotype <span class="math inline">\(y\)</span>.</p>
<span class="math display" id="eq:assoclmm">\[\begin{equation}
  y = X \beta + \sum_{k=1}^{m}{r_k} + e
\tag{1.1}
\end{equation}\]</span>
<p>where <span class="math inline">\(n\)</span> is the number of individuals, <span class="math inline">\(p\)</span> is the number of covariates or fixed effects, <span class="math inline">\(m\)</span> is the number of structured random effects apart from the residuals errors, <span class="math inline">\(y\)</span> is a phenotype vector of length <span class="math inline">\(n\)</span>, <span class="math inline">\(X\)</span> is a matrix of covariates of size <span class="math inline">\(n \times p\)</span>, <span class="math inline">\(\beta\)</span> is a vector of fixed effects of length <span class="math inline">\(p\)</span>. The vectors of random effects <span class="math inline">\(r_k\)</span> and <span class="math inline">\(e\)</span> are mutually uncorrelated and multivariate normally distributed as <span class="math inline">\(\mathcal{N}(0, \sigma^2_k R_k)\)</span> and <span class="math inline">\(\mathcal{N}(0, \sigma^2_r I)\)</span>. The variance-covariance matrices are parametrized with scalar parameters (referred as variance components) and constant matrices of size <span class="math inline">\(n \times n\)</span> that express relationships among <span class="math inline">\(n\)</span> individuals. The first <span class="math inline">\(m\)</span> random effects <span class="math inline">\(r_k\)</span> are referred here as structured, whereas the last component <span class="math inline">\(e\)</span> is simply the residual errors which are independent and identically distributed.</p>
<p>Thus, the phenotype follows a multivariate normal distribution (MVN) and Equation <a href="manuscript.html#eq:assoclmm">(1.1)</a> can be rewritten:</p>
<span class="math display" id="eq:assocmvn">\[\begin{equation}
  y \sim \mathcal{N} (X \beta, V) = \mathcal{N} (X \beta, \sum_{k=1}^{m}{\sigma_k^2 R_k} + \sigma_r^2 I) 
\tag{1.2}
\end{equation}\]</span>
<p>An assosiation test for a given variable in the matrix <span class="math inline">\(X\)</span> consists in constructing the score test statistic based on the estimates of effect size and its variance, <span class="math inline">\(Z = \hat{\beta}_x / \sqrt{var(\hat{\beta}_x)}\)</span>. The score follows the standard normal distribution <span class="math inline">\(Z \sim \mathcal{N}(0, 1)\)</span>, and the <span class="math inline">\(\chi^2\)</span> test with non-centrality parameter <span class="math inline">\(NCP = \beta^2_x / var(\hat{\beta}_x)\)</span> quantifies the statistical power for a given true effect size <span class="math inline">\(\beta_x\)</span>.</p>
<p>We further consider several parameterizations of the model in Equation <a href="manuscript.html#eq:assocmvn">(1.2)</a> that depend on (i) whether marginal genetic or gene-environment interaction effect is under testing; (ii) whether structured random effects are included or only the residual errors are present. The detailed derivation of the formulas presented next is given in Supplementary Material, Section <a href="supplementary-material.html#derivations">2.2</a>.</p>
<p>We introduce common assumptions and notations before going further. We assume that all vectors of the phenotype (<span class="math inline">\(y\)</span>) and covariates (columns in the matrix <span class="math inline">\(X\)</span>) are centered. The phenotype vector is additionally standardized (<span class="math inline">\(var(y) = 1\)</span>). The genotype vector <span class="math inline">\(x_g\)</span> is considered as a realization of a vector of random variables <span class="math inline">\(\mathcal{X_g}\)</span>, which is a genotype in <span class="math inline">\(n\)</span> individuals with a minor allele frequency <span class="math inline">\(p\)</span>. We denote the distribution <span class="math inline">\(\mathcal{X_g} \sim (\mu_g, \Sigma_g) = (p 1_n, \delta_g^2 K) = (p 1_n, 2 p (1-p) K)\)</span>, where <span class="math inline">\(K\)</span> is the kinship matrix of size <span class="math inline">\(n \times n\)</span> (it can be the identity matrix <span class="math inline">\(I\)</span> for genetically unrelated individuals) and <span class="math inline">\(1_n\)</span> is a vector of <span class="math inline">\(n\)</span> ones.</p>
</div>
<div id="testing-marginal-genetic-effect" class="section level3">
<h3><span class="header-section-number">1.2.2</span> Testing marginal genetic effect</h3>
<p>The genetic effect on phenotype in unrelated individuals is evaluated under the standard linear model <span class="math inline">\(y \sim \mathcal{N} (\mu x_0 + \beta_g x_g, \sigma_r^2 I)\)</span>, where <span class="math inline">\(x_0 = 1_n\)</span> is a vector of <span class="math inline">\(n\)</span> ones, <span class="math inline">\(\mu\)</span> is a mean of the phenotypic values, <span class="math inline">\(x_g\)</span> is a vector of length of the genotypic values, <span class="math inline">\(\beta_g\)</span> is the effect size of the genotype. The NCP parameter of the test in linear model is well known to be proportional to the sample size and the variance captured by the genotype (see also Section <a href="supplementary-material.html#lmg">2.2.1</a>).</p>
<span class="math display" id="eq:ncpgun">\[\begin{equation}
NCP_{unrel} \approx \beta_g^2 \delta_g^2 n  = \hat{\beta}_g^2 2 p (1 - p) n
\tag{1.3}
\end{equation}\]</span>
<p>When the individuals are genetically related and/or the covariance of the phenotype is modeled using structured relationship matrices among individuals, the following linear mixed model is stated, <span class="math inline">\(y \sim \mathcal{N} (\mu x_0 + \beta_g x_g, \sum_{k=1}^{m}{\sigma_k^2 R_k} + \sigma_r^2 I)\)</span>. The initial step in solving a linear mixed model is to estimate random effects parameters (<span class="math inline">\(\sigma_k^2\)</span> and <span class="math inline">\(\sigma_r^2\)</span>) by restricted maximum likelihood (REML) or other optimization technique <span class="citation">(Lynch and Walsh <a href="#ref-Lynch1998">1998</a>)</span>. Once the estimate of the variance-covariance matrix is found, <span class="math inline">\(\hat{V} = \sum{\hat{\sigma}_i^2 R_i} + \hat{\sigma}_r^2 I\)</span>, the generalized least squares (GLS) for fixed effects are applied in the following matrix form, <span class="math inline">\(\hat{\beta} = \left(X^T \hat{V}^{-1} X\right)^{-1} X \hat{V}^{-1} y\)</span>.</p>
<p>In Section <a href="supplementary-material.html#lmmg">2.2.2</a> we derived the estimate of <span class="math inline">\(\beta_g\)</span> and its variance, <span class="math inline">\(\hat{\beta}_g = \left(\tilde{x}_g^T \hat{V}^{-1} \tilde{x}_g\right)^{-1} \tilde{x}_g^T \hat{V}^{-1} \tilde{y}\)</span> and <span class="math inline">\(var(\hat{\beta_g}) = 1 / (\tilde{x}_g^T \hat{V}^{-1} \tilde{x}_g)\)</span>, respectively. We further approximated the term <span class="math inline">\(\tilde{x}_g^T \hat{V}^{-1} \tilde{x}_g\)</span> using the expected mean of a quadratic form of the random variable <span class="math inline">\(\mathcal{\tilde{X}_g}\)</span> and the transformation matrix <span class="math inline">\(\hat{V}^{-1}\)</span> (see Equation <a href="supplementary-material.html#eq:quadform1">(2.1)</a>). The NCP parameter of the test in linear mixed model has the following form, where <span class="math inline">\(tr\)</span> denote the trace operator.</p>
<span class="math display" id="eq:ncpgrel">\[\begin{equation}
NCP_{rel} \approx beta_g^2 tr(\hat{V}^{-1} \Sigma_g) = \beta_g^2 \delta_g^2 tr(\hat{V}^{-1} K) = \beta_g^2 2 p (1 - p) tr(\hat{V}^{-1} K)
\tag{1.4}
\end{equation}\]</span>
<p>The effective size multiplier, defined as <span class="math inline">\(NCP_{rel} / NCP_{unrel}\)</span>, gives a quantitative assessment of gain or loss in power when comparing the study design of related and unrelated individuals.</p>
<span class="math display" id="eq:ncpg">\[\begin{equation}
NCP_{rel} / NCP_{unrel} = tr(\hat{V}^{-1} K) / n
\tag{1.5}
\end{equation}\]</span>
<p>We further expand Equation <a href="manuscript.html#eq:ncpgrel">(1.4)</a> for two specific cases of (i) related individuals in families; (ii) unrelated individuals under the the infinite-testimal model. We also make use of the connection between the trace operator and eigen-value decomposition (Section <a href="supplementary-material.html#propositions">2.1</a>).</p>
<p>For individuals in families and the model <span class="math inline">\(y \sim \mathcal{N} (\mu x_0 + \beta_g x_g, \sigma_k^2 K + \sigma_r^2 I)\)</span>, we have an updated formula of <span class="math inline">\(NCP_{ref}\)</span>.</p>
<span class="math display" id="eq:ncpgfam">\[\begin{equation}
\begin{split}
NCP_{fam} &amp; = \beta_g^2 2 p (1 - p) tr \left( \left(\hat{\sigma}_k^2 K + \hat{\sigma}_r^2 I\right)^{-1} K \right)  \\
 &amp; = \beta_g^2 2 p (1 - p) tr \left( \left(\hat{\sigma}_k^2 I + \hat{\sigma}_r^2 K^{-1}\right)^{-1} \right) \\
 &amp; = \beta_g^2 2 p (1 - p) \sum_{i=1}^{n}{\left(\hat{\sigma}_k^2 + \hat{\sigma}_r^2 \left(\lambda_{K}^i\right)^{-1}\right)^{-1}}
\end{split}
\tag{1.6}
\end{equation}\]</span>
<p>When modeling the polygenic effect in unrelated individuals using the genetic relationship matrix (GRM) (denoted as <span class="math inline">\(M\)</span> in equations) <span class="math inline">\(y \sim \mathcal{N} (\mu x_0 + \beta_g x_g, \sigma_m^2 M + \sigma_r^2 I)\)</span>, we rewrite <span class="math inline">\(NCP_{ref}\)</span> as following.</p>
<span class="math display" id="eq:ncpggrm">\[\begin{equation}
\begin{split}
NCP_{unrel+grm} &amp; = \beta_g^2 2 p (1 - p) tr( (\hat{\sigma}_k^2 M + \hat{\sigma}_r^2 I)^{-1} )  \\
 &amp; = \beta_g^2 2 p (1 - p) \sum_{i=1}^{n}{(\hat{\sigma}_m^2 \lambda_{M}^i + \hat{\sigma}_r^2)^{-1}}
\end{split}
\tag{1.7}
\end{equation}\]</span>
</div>
<div id="testing-gene-environment-interaction-effect" class="section level3">
<h3><span class="header-section-number">1.2.3</span> Testing gene-environment interaction effect</h3>
<p>The gene-environment interaction effect on phenotype in unrelated individuals is evaluated under the standard linear model <span class="math inline">\(y \sim \mathcal{N} (\mu x_0 + \beta_g x_g + \beta_e x_e + \beta_{ge} x_{ge}, \sigma_r^2 I)\)</span>, where <span class="math inline">\(x_0 = 1_n\)</span> is a vector of <span class="math inline">\(n\)</span> ones, <span class="math inline">\(\mu\)</span> is a mean of the phenotypic values, <span class="math inline">\(x_g\)</span> is a vector of length of the genotypic values, <span class="math inline">\(\beta_g\)</span> is the effect size of the genotype, <span class="math inline">\(x_e\)</span> is a environment exposure vector of length <span class="math inline">\(n\)</span>, <span class="math inline">\(\beta_e\)</span> is the effect size of the environment exposure, <span class="math inline">\(x_{ge}\)</span> is a vector of length <span class="math inline">\(n\)</span> of gene-environment interaction, <span class="math inline">\(\beta_{ge}\)</span> is the interaction effect size.</p>
<p>The coding scheme of the genotypic and environmental variables to study gene-environment interaction under linear model is important and has been reviewed elsewhere <span class="citation">(Aschard <a href="#ref-Aschard2016">2016</a>)</span>. Here, we work with centered variables <span class="math inline">\(\tilde{x}_g\)</span> and <span class="math inline">\(\tilde{x}_e\)</span> and define the interaction variable <span class="math inline">\(\tilde{x}_{ge}\)</span> by (i) element-wise multiplication of the two variables, (ii) centering the resulted product. Once the covariates are centered as described above, the effect sizes and their standard errors can be estimated independently from other covariates if we assume that the two random variables of genotype and environmental exposure are generated independently <span class="citation">(Aschard <a href="#ref-Aschard2016">2016</a>, Appendix C)</span>. We also note that different coding schemes give different estimates of effect sizes, but the test statistic for gene-environment interaction (<span class="math inline">\(NCP\)</span>) is the same <span class="citation">(Aschard <a href="#ref-Aschard2016">2016</a>, Appendix B)</span>.</p>
<p>We first need to define two matrices <span class="math inline">\(E\)</span> and <span class="math inline">\(D\)</span>. The matrix <span class="math inline">\(E\)</span> is related to the environment exposure (centered) vector <span class="math inline">\(\tilde{x}_e\)</span>: <span class="math inline">\(E\)</span> is the diagonal matrix with values equal to those observed in <span class="math inline">\(\tilde{x}_e\)</span>, i.e. <span class="math inline">\(diag(E) = \tilde{x}_e\)</span>. We also introduce a matrix <span class="math inline">\(D\)</span>, which value at row <span class="math inline">\(i\)</span> and column <span class="math inline">\(j\)</span> is equal to the product of two diagonal entries <span class="math inline">\(i\)</span> and <span class="math inline">\(j\)</span> of <span class="math inline">\(E\)</span>, i.e. <span class="math inline">\(D_{i,j} = E_{i,i} E_{j,j}\)</span>. When the environmental exposure is binary and the observed frequency of exposure is <span class="math inline">\(f\)</span>, then we denote the matrices as <span class="math inline">\(E_b\)</span> and <span class="math inline">\(D_b\)</span>. Then the values on diagonal of the matrix <span class="math inline">\(E_b\)</span> are either <span class="math inline">\(-f\)</span> or <span class="math inline">\(1 - f\)</span>, while values of the matrix <span class="math inline">\(D_b\)</span> are either <span class="math inline">\(f^2\)</span>, <span class="math inline">\((1 - f)^2\)</span> or <span class="math inline">\(f (1 - f)\)</span>.</p>
<p>Then the NCP parameter of the test in linear model has the following form (see Section <a href="supplementary-material.html#lmmge">2.2.5</a>).</p>
<span class="math display" id="eq:ncpgeun">\[\begin{equation}
NCP_{unrel}^i \approx \beta_{ge}^2 \delta_g^2 tr(E_b^2) = \beta_{ge}^2 2 p (1 - p) f (1 - f) n 
\tag{1.8}
\end{equation}\]</span>
<p>When the individuals are genetically related and/or relationships among individuals are modeled, we again use the linear mixed model <span class="math inline">\(y \sim \mathcal{N} (\mu x_0 + \beta_g x_g + \beta_e x_e + \beta_{ge} x_{ge}, \sum_{k=1}^{m}{\sigma_k^2 R_k} + \sigma_r^2 I)\)</span>.</p>
<p>In Section <a href="supplementary-material.html#lmmge">2.2.5</a> we derived the estimate of <span class="math inline">\(\beta_{ge}\)</span> and its variance for linear mixed model, <span class="math inline">\(\hat{\beta}_{ge} = \left(\tilde{x}_{ge}^T \hat{V}^{-1} \tilde{x}_{ge}\right)^{-1} \tilde{x}_{ge}^T \hat{V}^{-1} \tilde{y}\)</span> and <span class="math inline">\(var(\hat{\beta_{ge}}) = 1 / (\tilde{x}_{ge}^T \hat{V}^{-1} \tilde{x}_{ge})\)</span>, respectively. As <span class="math inline">\(\tilde{x}_{ge}\)</span> is a realization of a random variable <span class="math inline">\(\mathcal{\tilde{X}}_{ge} = E \mathcal{\tilde{X}}_{g}\)</span>, we showed that <span class="math inline">\(var(\mathcal{\tilde{X}}_{ge}) = \delta_g^2 D \circ K\)</span>.</p>
<p>Here, we introduce a special kinship matrix <span class="math inline">\(K_{D}\)</span> “masked” by the (observed) environmental exposure though the matrix <span class="math inline">\(D\)</span> (the operator <span class="math inline">\(\circ\)</span> denotes the Hadamard product, i.e. the element-wise multiplication).</p>
<span class="math display" id="eq:kdmat">\[\begin{equation}
\begin{split}
K_{D} =  D \circ K
\end{split}
\tag{1.9}
\end{equation}\]</span>
<p>In Section <a href="supplementary-material.html#lmmge">2.2.5</a> we further approximated the term <span class="math inline">\(\tilde{x}_{ge}^T \hat{V}^{-1} \tilde{x}_{ge}\)</span> by applying the expression for the mean of a quadratic form of the random variable <span class="math inline">\(\mathcal{\tilde{X}_{ge}}\)</span> and the transformation matrix <span class="math inline">\(\hat{V}^{-1}\)</span> (see Equation <a href="supplementary-material.html#eq:quadform1">(2.1)</a>). The NCP parameter of the test in linear mixed model has the following form.</p>
<span class="math display" id="eq:ncpgerel">\[\begin{equation}
NCP_{rel}^i \approx \beta_{ge}^2 \delta_g^2 tr(\hat{V}^{-1} K_{D}) = \beta_{ge}^2 2 p (1 - p) tr(\hat{V}^{-1} K_{D})
\tag{1.10}
\end{equation}\]</span>
<p>The <span class="math inline">\(K_{D}\)</span> matrix is equal to the <span class="math inline">\(E^2\)</span> matrix in the case of genetically unrelated individuals (<span class="math inline">\(K = I\)</span>), and the two formulas <a href="manuscript.html#eq:ncpgerel">(1.10)</a> and <a href="manuscript.html#eq:ncpgeun">(1.8)</a> become the same. We also note that the variance of the environmental exposure is contained within the matrices <span class="math inline">\(K_{D}\)</span> and <span class="math inline">\(E^2\)</span>, although it is possible to similarly define the scaled matrices.</p>
<p>TODO: <span class="math inline">\(V = \sigma^2_k K + \sigma^2_i K^i + \sigma^2_r I\)</span> <span class="citation">(Sul et al. <a href="#ref-Sul2016">2016</a>)</span></p>
</div>
</div>
<div id="results" class="section level2">
<h2><span class="header-section-number">1.3</span> Results</h2>
<div id="analytical-results-for-testing-marginal-genetic-effect" class="section level3">
<h3><span class="header-section-number">1.3.1</span> Analytical results for testing marginal genetic effect</h3>
<table>
<caption><span id="tab:assoc-gen">Table 1.1: </span> Analytical comparison of study designs to detect marginal genetic association. Study designs differ in individual relationships that informs modeling of outcome (<span class="math inline">\(y\)</span>) and distribution of genotype under association test (<span class="math inline">\(x_g\)</span>). Study designs under comparison include: unrelated individuals; related individuals in families; unrelated individuals with a grouping factor such as house-hold (not related to a variable under test). Notation: <span class="math inline">\(\tilde{x_g}\)</span>, mean-centered genotype vector <span class="math inline">\(x_g\)</span>; <span class="math inline">\(\delta_g^2 = 2 p (1 - p)\)</span>, the variance of genotype random variable with the minor allele frequency <span class="math inline">\(p\)</span>; <span class="math inline">\(K\)</span>, the additive kinship matrix for family-based study design; <span class="math inline">\(NCP\)</span>, the non-centrality parameter of the test; <span class="math inline">\(\hat{V}\)</span>, the estimated variance-covariance matrix of <span class="math inline">\(y\)</span>.</caption>
<colgroup>
<col width="23%" />
<col width="16%" />
<col width="17%" />
<col width="41%" />
</colgroup>
<thead>
<tr class="header">
<th>Study design</th>
<th><span class="math inline">\(V = Var(y)\)</span></th>
<th><span class="math inline">\(\Sigma_g = Var(\mathcal{\tilde{X}}_g)\)</span></th>
<th><span class="math inline">\(NCP\)</span></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Unrelated</td>
<td><span class="math inline">\(\sigma_r^2 I\)</span></td>
<td><span class="math inline">\(\delta_g^2 I\)</span></td>
<td><span class="math inline">\(\beta^2_g (\tilde{x_g}^T \tilde{x_g}) \approx \beta^2_g \delta_g^2 n\)</span></td>
</tr>
<tr class="even">
<td>Families</td>
<td><span class="math inline">\(\sigma_k^2 K + \sigma_r^2 I\)</span></td>
<td><span class="math inline">\(\delta_g^2 K\)</span></td>
<td><span class="math inline">\(\beta^2_g (\tilde{x_g}^T \hat{V}^{-1} \tilde{x_g}) \approx \beta^2_g \delta_g^2 \mbox { } tr(\hat{V}^{-1} K)\)</span></td>
</tr>
<tr class="odd">
<td>Unrelated + Grouping</td>
<td><span class="math inline">\(\sigma_h^2 H + \sigma_r^2 I\)</span></td>
<td><span class="math inline">\(\delta_g^2 I\)</span></td>
<td><span class="math inline">\(\beta^2_g (\tilde{x_g}^T \hat{V}^{-1} \tilde{x_g}) \approx \beta^2_g \delta_g^2 \mbox { } tr(\hat{V}^{-1})\)</span></td>
</tr>
</tbody>
</table>
<div class="figure" style="text-align: center"><span id="fig:power-marginal"></span>
<img src="figures/07-figure-power-marginal-two-panels.png" alt="Three study designs are compared in terms of power to detect marginal genetic effect under the model \(y \sim \mathcal{N}(\mu + \beta_g x_g, V)\) (see also Table 1.1). The reference study design “Unrelated” with \(V = \sigma^2_r I\) is; the study" width="75%" />
<p class="caption">
Figure 1.1: Three study designs are compared in terms of power to detect marginal genetic effect under the model <span class="math inline">\(y \sim \mathcal{N}(\mu + \beta_g x_g, V)\)</span> (see also Table <a href="manuscript.html#tab:assoc-gen">1.1</a>). The reference study design “Unrelated” with <span class="math inline">\(V = \sigma^2_r I\)</span> is; the study
</p>
</div>

<p>Two study designs with a structured variance components, unrelated individuals with a non-genetic grouping factor and related individuals in families, are compared to the reference study design of unrelated individuals. The performance is evaluated to To make the study designs comparable, the sum of variance components in the <span class="math inline">\(V\)</span> matrix is equal to one. (a) The effective size multiplier, estimated as <span class="math inline">\(tr(V^{-1} \Sigma_g) / n\)</span>, depends on the variance explained. (b) When the variance explained fixed to 50% and the sample size varies, Notation: <span class="math inline">\(n\)</span>, the sample size.</p>
<p>Ref <a href="manuscript.html#fig:power-marginal">1.1</a></p>
</div>
<div id="analytical-results-for-testing-gene-environment-interaction-effect" class="section level3">
<h3><span class="header-section-number">1.3.2</span> Analytical results for testing gene-environment interaction effect</h3>
<table>
<caption><span id="tab:assoc-ge">Table 1.2: </span> Analytical comparison of study designs to detect gene-environment interaction association.</caption>
<colgroup>
<col width="16%" />
<col width="18%" />
<col width="19%" />
<col width="45%" />
</colgroup>
<thead>
<tr class="header">
<th>Study design</th>
<th><span class="math inline">\(V = Var(y)\)</span></th>
<th><span class="math inline">\(\Sigma_{ge} = Var(E \mathcal{\tilde{X}}_g)\)</span></th>
<th><span class="math inline">\(NCP\)</span></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Unrelated</td>
<td><span class="math inline">\(\sigma_r^2 I\)</span></td>
<td><span class="math inline">\(\delta_g^2 E^2\)</span></td>
<td><span class="math inline">\(\beta^2_{ge} (\tilde{x_{ge}}^T \tilde{x_{ge}}) \approx \beta^2_{ge} \delta_g^2 tr(E^2)\)</span></td>
</tr>
<tr class="even">
<td>Unrelated (binary)</td>
<td><span class="math inline">\(\sigma_r^2 I\)</span></td>
<td><span class="math inline">\(\delta_g^2 E_b^2\)</span></td>
<td><span class="math inline">\(\beta^2_{ge} (\tilde{x_{ge}}^T \tilde{x_{ge}}) \approx \beta^2_{ge} \delta_g^2 f (1 -f) n\)</span></td>
</tr>
<tr class="odd">
<td>Families</td>
<td><span class="math inline">\(\sigma_k^2 K + \sigma_i^2 K_i + \sigma_r^2 I\)</span></td>
<td><span class="math inline">\(\delta_g^2 K_D\)</span></td>
<td><span class="math inline">\(\beta^2_{ge} (\tilde{x_{ge}}^T \hat{V}^{-1} \tilde{x_{ge}}) \approx \beta^2_{ge} \delta_g^2 \mbox { } tr(\hat{V}^{-1} K_D)\)</span></td>
</tr>
</tbody>
</table>
<div class="figure" style="text-align: center"><span id="fig:power-interaction"></span>
<img src="figures/09-figure-power-interaction-expsib-two-panels.png" alt="Comparison of study designs to detect gene-environment interaction effect. The binary exposure is generated such a way that siblings are exposed, while parents don’t (the frequency of exposure \(f = 3/5 = 0.6\))." width="75%" />
<p class="caption">
Figure 1.2: Comparison of study designs to detect gene-environment interaction effect. The binary exposure is generated such a way that siblings are exposed, while parents don’t (the frequency of exposure <span class="math inline">\(f = 3/5 = 0.6\)</span>).
</p>
</div>

<p>Ref <a href="manuscript.html#fig:power-interaction">1.2</a></p>
</div>
<div id="implication-in-association-studies" class="section level3">
<h3><span class="header-section-number">1.3.3</span> Implication in association studies</h3>
<div id="effective-sample-size-in-post-gwas-analyses" class="section level4">
<h4><span class="header-section-number">1.3.3.1</span> Effective sample size in post-GWAS analyses</h4>
<p>Many post-GWAS analyses, such as LDSC regresssion or meta-analysis, rely on the sample size as an input parameter under assumption that summary statistics come from a set of unrelated individuals under association linear model. Though, tools that employ linear mixed models, for example, BOLT-LMM, achieve the effective sample size (<span class="math inline">\(n_{eff}\)</span>) larger than the true sample size (<span class="math inline">\(n\)</span>) <span class="citation">(P.-R. Loh et al. <a href="#ref-loh2018mixed">2018</a>)</span>.</p>
<p>Gazal <em>et al.</em> showed that the LDSC regression overestimates per-variant heritability when using summary statistics from BOLT-LMM and the true sample size (<span class="math inline">\(n\)</span>) <span class="citation">(Gazal et al. <a href="#ref-Gazal2017">2017</a>)</span>. The authors proposed an empirical solution to estimate the scaling factor or the effective size multiplier: taking ratios of chi-squared statistics computed by BOLT-LMM <em>vs.</em> linear regression at genome-wide significant variants <span class="citation">(P.-R. Loh et al. <a href="#ref-loh2018mixed">2018</a>)</span>. In our work, we derived an analytical form for the effective size multiplier in Equations <a href="manuscript.html#eq:ncpg">(1.5)</a> and <a href="manuscript.html#eq:ncpggrm">(1.7)</a>.</p>
</div>
<div id="optimization-of-study-design" class="section level4">
<h4><span class="header-section-number">1.3.3.2</span> Optimization of study design</h4>
<p>In genetic association and linkage studies performed in related individuals grouped in families, selection of a family-based design that achieves the most statistical power has been extensively studied [ref]. For example, the multipoint linkage analysis was shown to have greater power in extended pedigrees than in smaller pedigrees such as sibships <span class="citation">(Almasy and Blangero <a href="#ref-Almasy1998">1998</a>)</span>.</p>

</div>
</div>
</div>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-loh2018mixed">
<p>Loh, Po-Ru, Gleb Kichaev, Steven Gazal, Armin P Schoech, and Alkes L Price. 2018. “Mixed-Model Association for Biobank-Scale Datasets.” <em>Nature Genetics</em>. Nature Publishing Group, 1.</p>
</div>
<div id="ref-young2016multiple">
<p>Young, Alexander I, Fabian Wauthier, and Peter Donnelly. 2016. “Multiple Novel Gene-by-Environment Interactions Modify the Effect of Fto Variants on Body Mass Index.” <em>Nature Communications</em> 7. Nature Publishing Group: 12724.</p>
</div>
<div id="ref-Lynch1998">
<p>Lynch, Michael, and Bruce Walsh. 1998. <em>Genetics and Analysis of Quantitative Traits</em>. Vol. 1. Sinauer Sunderland.</p>
</div>
<div id="ref-Aschard2016">
<p>Aschard, Hugues. 2016. “A perspective on interaction effects in genetic association studies.” <em>Genetic Epidemiology</em> 40 (8): 678–88.</p>
</div>
<div id="ref-Sul2016">
<p>Sul, Jae Hoon, Michael Bilow, Wen-Yun Yang, Emrah Kostem, Nick Furlotte, Dan He, and Eleazar Eskin. 2016. “Accounting for Population Structure in Gene-by-Environment Interactions in Genome-Wide Association Studies Using Mixed Models.” <em>PLoS Genetics</em> 12 (3). Public Library of Science: e1005849.</p>
</div>
<div id="ref-Gazal2017">
<p>Gazal, Steven, Hilary K. Finucane, Nicholas A. Furlotte, Po Ru Loh, Pier Francesco Palamara, Xuanyao Liu, Armin Schoech, et al. 2017. “Linkage disequilibrium-dependent architecture of human complex traits shows action of negative selection.” <em>Nature Genetics</em> 49 (10). Nature Publishing Group: 1421–7. doi:<a href="https://doi.org/10.1038/ng.3954">10.1038/ng.3954</a>.</p>
</div>
<div id="ref-Almasy1998">
<p>Almasy, L, and J Blangero. 1998. “Multipoint quantitative-trait linkage analysis in general pedigrees.” <em>American Journal of Human Genetics</em> 62 (5): 1198–1211. doi:<a href="https://doi.org/10.1086/301844">10.1086/301844</a>.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="index.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="supplementary-material.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"linkedin": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"download": ["gxefam.pdf"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "";
    if (src === "" || src === "true") src = "https://cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(src))
      src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
